Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroGraphics/Driver/MicroGraphics.cs

```diff
@@ -19,7 +19,15 @@ public partial class MicroGraphics
         /// <summary>
         /// PixelBuffer draw target
         /// </summary>
-        protected IPixelBuffer pixelBuffer;
+        protected IPixelBuffer PixelBuffer
+        {
+            get
+            {
+                if (display != null) return display.PixelBuffer;
+                return _memoryBuffer;
+            }
+        }
+        private readonly IPixelBuffer _memoryBuffer;
 
         /// <summary>
         /// Ingore pixels that are outside of the pixel buffer coordinate space
```

Comment:
Ignore

Reply:
haha .. I'll do a global search for ingore :)